newswires/client/src/dateHelpers.ts

```diff
@@ -1,11 +1,22 @@
 import dateMath from '@elastic/datemath';
-import moment from 'moment';
+import moment from 'moment-timezone';
 
 export interface TimeRange {
 	start: string;
 	end: string;
 }
 
+export const convertToLocalDate = (timestamp: string): string => {
+	const cleanedTimestamp = timestamp.replace(/\[.*]$/, '');
```

Comment:
If I understand the regex correctly then we're using it to discard the timezone information from the timestamp. But this function will only produce the correct local time if the `timestamp` is UTC, won't it? In which case, would it be better to replace `\[Etc\/UTC]$` instead of `\[.*]$`, because we probably want the function to error if we pass a non-UTC timestamp to it.